### PR TITLE
Milestone: Stats

### PR DESCRIPTION
**Pull request by ECWireless**
- [x]  Add view
- [x]  Add container
- [x]  Add header
- [x]  Add info cards
- [x]  Add graph
- [x]  Make graph dynamic
- [x]  Allow fullscreen graph

### CODE REVIEW

File: src/assets/icons/diamond.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...62 6.087h-1.303v1.522h1.303V6.087z" fill="#8ED4E6"/><path d="M18.26 6.087h-1.303v1.522h1.303V6.087zM16.957 6.087h-1.305v1.522h1.305V6.087z" fill="#8ED4E6"/><path d="M15.65 6.087h-1.302v1.522h1.302V6.087zM14.348 6.087h-1.305v1.522h1.305V6.087zM13.04 6.087h-1.302v1.522h1.302V6.087zM11.738 6.087h-1.302v1.522h1.302V6.087zM10.436 6.087H9.13v1.522h1.305V6.087z" fill="#67CBDF"/><path d="M19.562 4.564h-1.303v1.523h1.303V4.564z" fill="#8ED4E6"/><path d="M18.26 4.564h-1.303v1.523h1.303V4.564zM16.957 4.564h-1.305v1.523h1.305V4.564z" fill="#8ED4E6"/><path d="M15.65 4.564h-1.302v1.523h1.302V4.564zM14.348 4.564h-1.305v1.523h1.305V4.564zM13.04 4.564h-1.302v1.523h1.302V4.564zM11.738 4.564h-1.302v1.523h1.302V4.564z" fill="#67CBDF"/><path d="M18.26 3.042h-1.303v1.522h1.303V3.042zM16.957 3.042h-1.305v1.522h1.305V3.042z" fill="#8ED4E6"/><path d="M15.65 3.042h-1.302v1.522h1.302V3.042zM14.348 3.042h-1.305v1.522h1.305V3.042zM13.04 3.042h-1.302v1.522h1.302V3.042z" fill="#67CBDF"/><path d="M16.957 1.52h-1.305v1.522h1.305V1.519z" fill="#8ED4E6"/><path d="M15.65 1.52h-1.302v1.522h1.302V1.519zM14.348 1.52h-1.305v1.522h1.305V1.519z" fill="#67CBDF"/><path d="M15.65 0h-1.302v1.52h1.302V0z" fill="#67CBDF"/></svg>
```
Comment (Evalir) [inline src/assets/icons/diamond.svg:1]: Are these optimized? :)

**Review by Evalir — APPROVED** [review]
Sweet!

File: src/assets/icons/diamond.svg (hunk 1 above)
Comment (ECWireless) [reply src/assets/icons/diamond.svg:1]: Yup, should be! All SVGs have been run though `svgo`'s optimizer. Or is there a different way they should be optimized?

Comment (Evalir) [reply src/assets/icons/diamond.svg:1]: Nope, was just checking haha

**Pull request merged**